components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/decorators.ts

```diff
@@ -10,42 +10,44 @@
  * SPDX-License-Identifier: EPL-2.0
  */
 
-import * as rs from "@dirigible/http/rs"
+import { rs, response, request } from "."
 
 const router = rs.service();
 let instance = null;
 
-export function Controller(ctr: {new()}, context: ClassDecoratorContext): void {
-    instance = new ctr();
-    router.execute();
-}
+export class Decorators {
+    public static Controller(ctr: {new()}, context: ClassDecoratorContext): void {
+        instance = new ctr();
+        router.execute();
+    }
 
-export const Get = createRequestDecorator("get")
-export const Post = createRequestDecorator("post")
-export const Put = createRequestDecorator("put")
-export const Patch = createRequestDecorator("patch")
-export const Delete = createRequestDecorator("delete")
-export const Head = createRequestDecorator("head")
-export const Options = createRequestDecorator("options")
+    public static Get = this.createRequestDecorator("get")
```

Comment:
For access to static methods of the class, you the class name instead of `this`:

```ts
public static Get = Decorators.createRequestDecorator("get");
```

Also for exported constants, the change should be as follows:

```ts
public static readonly Get = Decorators.createRequestDecorator("get");
```